pkg/util/crypto/checksum.go

```diff
@@ -0,0 +1,20 @@
+package crypto
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"hash/crc32"
+)
+
+func ChecksumString(data []byte) string {
+	// Calculate the checksum with crc32 IEEE
+	checksum := crc32.ChecksumIEEE(data)
+	// Turn the 32-bit unsigned checksum into 4 bytes in big endian order.
+	byteSlice := make([]byte, 4)
+	byteSlice = binary.BigEndian.AppendUint32(byteSlice, checksum)
+
+	// Encode the 4 bytes in hex format.
+	checksumString := hex.EncodeToString(byteSlice)
+
+  return checksumString
```

Comment:
It seems that your editor is not configured properly to edit Go source files. I see you sometimes mess up with the indentation. Do you need help in setting up the editor?

Reply:
I just disabled auto format :sosad:
Let me turn it on.